Tests/Specs/versioned-api/tests/crud-api-version-1-strict.json

```diff
@@ -651,7 +651,7 @@
       ]
     },
     {
-      "description": "find command with declared API version appends to the command, but getMore does not",
```

Comment:
the old version of this test was failing, so syncing was required. however, the test file as written was schema version 1.4 and required support for your serverless runOn requirement. I didn't want to re-implement and conflict with whatever you've done for that, so I just manually edited this file to only include the change to this test case. I assume as part of the serverless PR you'll need to bring in the updated versions of these files anyway(?)

Reply:
Yep, after I rebase off of this patch I'll just make sure these tests are synced up.